.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    // Allow `rust-analyzer` and `cargo` to run simultaneously.
+    // This uses a heavy amount of storage space, so consider commenting it out.
+    "rust-analyzer.cargo.targetDir": true
+}
```

Comment:
Do you have the editorconfig plugin enabled? There should be a terminating newline inserted automatically here

Reply:
Nope! Installing.